source/appModules/winword.py

```diff
@@ -11,10 +11,21 @@
 import appModuleHandler
 from scriptHandler import script
 import ui
+from logHandler import log
 from NVDAObjects.IAccessible.winword import WordDocument as IAccessibleWordDocument
 from NVDAObjects.UIA.wordDocument import WordDocument as UIAWordDocument
 from NVDAObjects.window.winword import WordDocument
 from utils.displayString import DisplayStringIntEnum
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+	import inputCore
+
+
+# From WdOutlineLevel enumeration
+# See https://learn.microsoft.com/en-us/office/vba/api/word.wdoutlinelevel
+class WdOutlineLevel(IntEnum):
```

Comment:
@SaschaCowley I have accepted your suggestion.

Though, thinking again to it, I should probably move this enum to `NVDAObjects/window/winword.py` since it is supported both in Word and Outlook. And because 10 (BODY_TEXT) is used in `_iterHeadings`.

I'll refactor my code.